src/cdk/overlay/position/overlay-position-builder.ts

```diff
@@ -20,14 +20,16 @@ import {
   FlexibleConnectedPositionStrategyOrigin,
 } from './flexible-connected-position-strategy';
 import {GlobalPositionStrategy} from './global-position-strategy';
+import { _CoalescedStyleScheduler } from './coalesced-style-scheduler';
 
 
 /** Builder for overlay position strategy. */
 @Injectable({providedIn: 'root'})
 export class OverlayPositionBuilder {
   constructor(
       private _viewportRuler: ViewportRuler, @Inject(DOCUMENT) private _document: any,
-      private _platform: Platform, private _overlayContainer: OverlayContainer) {}
+      private _platform: Platform, private _overlayContainer: OverlayContainer,
+      @Inject(_CoalescedStyleScheduler) private _scheduler: _CoalescedStyleScheduler) {}
```

Comment:
_CoalescedStyleScheduler should already be injectable, so you don't need to add @Inject